modules/client/src/interfaces.ts

```diff
@@ -18,7 +19,7 @@ export interface IClientMethods extends IClientCore {
   pinMetadata: (params: DaoMetadata) => Promise<string>;
   /** Retrieves the asset balances of the given DAO, by default, ETH, DAI, USDC and USDT on Mainnet*/
   getDaoBalances: (
-    daoAddressOrEns: string,
+    params: AssetBalanceQueryParams,
```

Comment:
`DaoBalancesQueryParams` maybe?  for future easy discoverability

Reply:
makes sense